protractor/e2e/integretion/second-page.spec.ts

```diff
@@ -55,9 +57,22 @@ describe('test for second page', () => {
         await secondPage.waitForElementVisible(secondPage.pdfContent);
     });
 
-    fit('info card error case', async () => {
+    it('info card error case', async () => {
         await secondPage.scrollToElement(secondPage.infoCardSection);
         await secondPage.waitForElementVisible(secondPage.infoEmailInpt);
         await secondPage.infoSubmitBtn.click();
     });
+
+    fit('should open page blog pages with articles', async () => {
+        await secondPage.waitForElementVisible(secondPage.blogHeaderLink);
+        await secondPage.blogHeaderLink.click();
+        await blogPage.waitForElementVisible(blogPage.postClass.first());
+        expect(await blogPage.getCurrentUrl()).toContain(blogPage.pageUrl);
+        expect(await blogPage.postClass.count()).toEqual(20);
+        await blogPage.scrollToElement(blogPage.paginationBlock);
+        await blogPage.secondBlogPageLink.click();
+        await secondPage.waitForElementVisible(blogPage.postClass.first());
+        expect(await blogPage.getCurrentUrl()).toContain('page/2');
+        expect(await blogPage.postClass.count()).toBeGreaterThan(1);
+    });
```

Comment:
It's advisable to divide all this logic into 2 it blocks:
The first one for checking "Navigate to blog and check articles, pages navigation"
And the second one: "Navigate to blog and check articles, pages navigation"

Reply:
Every scenario was moved to sepeparete spec  blog-page.spec.ts with the latest commit, here is outdated code